### DISCUSSION TITLE
How best to specify species for stable cell lines and other cultured samples

### DISCUSSION THREAD

**Discussion by philbrooksIII** [dandi/helpdesk#89] [Q&A]
DANDI currently requires a binomial name or NCBI taxon identifier to specify the species for each subject. It's not clear how this should be applied to samples such as, for example, the HEK293T stable cell line. While human derived, this cell line has a decidedly strange genome (hypotriploid) with non-human genetic elements added during cell line generation. Does it make the most sense to just specify species as 'homo sapiens', then specifying the cell line in the strain field, or would it make more sense to index this (and other) stable cell line as its own entity. 
Would perhaps allowing links to other ontobee entries other than the NCBI taxa be one possible solution? (See entry for HEK293T in ontobee:BTO: https://ontobee.org/ontology/BTO?iri=http://purl.obolibrary.org/obo/BTO_0002181)?

Maybe this is a solved problem for those with more experience classifying cell lines, but I don't know the answer.

Thanks

**Comment by satra** [comment]
@patrick-lloyd-ray, @saskiad - do you have any thoughts on how to model this?

**Comment by patrick-lloyd-ray** [comment]
> Does it make the most sense to just specify species as 'homo sapiens', then specifying the cell line in the strain field

I think this is the solution that makes most sense to me, although I've not dealt directly with these types of cell lines so would welcome input from others who have.
